velox/functions/prestosql/StringFunctions.h

```diff
@@ -15,6 +15,8 @@
  */
 #pragma once
 
+#include <libstemmer.h>
```

Comment:
Let's create a separate file for this function.

Reply:
create a new file named: `WordStem.h`